src/utils/index.js

```diff
@@ -27,3 +27,9 @@ export const getPathFromLiteralTag = (strings, expressions) =>
       `${result}${currentString}${expressions[i] ? expressions[i] : ""}`,
     ""
   );
+
+export const warn = (conditional, description, warningKey) => {
+  if (process.env.NODE_ENV !== "production" && conditional) {
+    console.warn(`useStyles${warningKey !== "" ? " "+warningKey : "" }: ${description}`);
```

Comment:
This should be something like this.

```suggestion
    console.warn(`useStyles ${warningKey ? warningKey : ''}: ${description} You are seeing this warning because you are in development mode. In a production build there will be no warning.`);
```

Reply:
Some warnings come with "no warning and these styles will be ignored." or sometimes "no warning and these constants will be ignored." so I pass the whole description as a single param only. (to avoid further params specifying if I will have 'ignoring' statement and its type).

Reply:
We can get rid of that `these styles will be ignored` part for now.